cloudflared/rootfs/etc/s6-overlay/scripts/cloudflared-config.sh

```diff
@@ -281,6 +281,7 @@ fi
 
 # Write log level to S6 environment
 printf "%s" "${log}" > /var/run/s6/container_environment/CLOUDFLARED_LOG
+CLOUDFLARED_LOG=${log}
```

Comment:
```suggestion
export CLOUDFLARED_LOG=${log}
```

For other processes to access it, it needs to be exported. Is it the case here?

Reply:
Never mind. It is not the case.

Reply:
Other processes should be fine using the s6 env variable from line 283 (otherwise the managed tunnel would also fail which isn't the case). Did you test with my changes? Otherwise we could also use your recommendation I guess it doesn't matter in this case

Reply:
Indeed it doesn't matter in this case.